test/NonSilo.Tests/General/InterfaceRules.cs

```diff
@@ -13,7 +13,7 @@ namespace UnitTests.General
 {
     #region simple interfaces
 
-    public interface ITestGrain_VoidMethod : IAddressable
+/*    public interface ITestGrain_VoidMethod : IAddressable
```

Comment:
Need to work out how to support these tests given we're performing code generation on this project - probably by moving these types to a non-codegen'd project or similar.